### PR TITLE
Makes vacuum sealer craftable.

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
There's not really any good reason that a vacuum sealer should be
impossible to craft when you can make so many other things that are much
more complex.

### CODE REVIEW

File: data/json/recipes/recipe_electronics.json [hunk 1]
```diff
@@ -2239,5 +2239,33 @@
...
+      [ "scrap", 4 ]
+    ],
+    [
+      [ "element", 2 ]
+    ],
+    [
+      [ "hose", 1 ]
+    ],
+    [
+      [ "power_supply", 1 ]
```
Comment (KA101) [inline data/json/recipes/recipe_electronics.json]: I'm thinking a motor or something to pump out the air, at least.  This was back in the day before we had piles of miscellaneous parts.

Comment (kevingranade) [inline data/json/recipes/recipe_electronics.json]: I'd like to push back on the ratcheting up of what is craftable.
Everything is not supposed to be craftable.  At the very least this needs
an uncraftable 'air pump' part that can be harvested somewhere, but
building gaskets/pistons etc out of scrap is a bit much.

Comment (Darkyhard) [inline data/json/recipes/recipe_electronics.json]: Well, a survivor can somehow make these out of scrap for pneumatic weapons which appear to use compressed air for firing, so why couldn't they make one for a vacuum sealer? However, I am not sure how you're supposed to make an air pump without at least a soldering iron or a welder.

Comment (Zireael07) [inline data/json/recipes/recipe_electronics.json]: Agreed, either dump the craftable pneumatic weapons or let the vacuum sealer in.

**Comment by Rivet-the-Zombie** [comment]
Added small electric motor, soldering iron, superglue to recipe.

As it turns out, making a vacuum pump from scrap is [far from impossible](http://sci-toys.com/scitoys/scitoys/aero/vacuum/).

**Comment by KA101** [comment]
>As it turns out, making a vacuum pump from scrap is far from impossible.

>**factory made** one-way valves, and that was the most expensive part of the first pump

(Emphasis added-KA101)

Sounds like a noncraftable part to me... :-(

**Comment by Rivet-the-Zombie** [comment]
Sounds like a noncraftable part to me... :-(

[Not really noncraftable.](http://mods-n-hacks.wonderhowto.com/how-to/make-super-cheap-one-way-check-valve-0147955/) I have a feeling they used a factory-made one so as to avoid having to explain how to make that part as well.

**Comment by KA101** [comment]
As to Kevin's point, those photos look fairly bulky given the whole setup.  I'd throw out that perhaps a crafted vacuum sealer might be a point or two larger, at least.

**Comment by Rivet-the-Zombie** [comment]
Alright, I can make us a crafted one. It's just that a vacuum pump and a pair of heating elements shouldn't be **impossible** to assemble together.

**Comment by Rivet-the-Zombie** [comment]
We've now got a *makeshift vacuum sealer* which uses double the batteries per operation.

I also restored the original deconstruction recipe for the standard model.

**Comment by KA101** [comment]
I'm OK with the makeshift.

File: data/json/recipes/recipe_electronics.json (hunk 1 above)
Comment (kevingranade) [inline data/json/recipes/recipe_electronics.json]: FYI, this argument is ignored every time it's made.  If you were going around complaining about what we are allowing you might have some credibility, but as it is it just looks like an excuse to always support things getting added, no matter how unrealistic.

**Comment by kevingranade** [comment]
Ideally it would be hand-pumped and take longer to do the recipe, but I don't think we have infrastructure for that so it works as is.

**Pull request merged**